Update create-kitties.md (#67)
substrate-developer-hub/substrate-how-to-guides: docs/09-tutorials/01-Kitties/create-kitties.md

### 1. Kitty struct scaffolding

> We added additional comments to the code from Part I to better assist you with the action items in this tutorial module By the end of Part I, you should have the code below in the file /pallets/mykitties/src/lib.rs:

Reviewer: ```suggestion
We added additional comments to the code from Part I to better assist you with the action items in this module. By the end of Part I, your `/pallets/mykitties/src/lib.rs` file should look like this:
```

Author: Looks good!
